test/input_files/v4.2/failed/failed_body_sample_010.vcf

```diff
@@ -1,5 +1,5 @@
 ##fileformat=VCFv4.2
-##CauseOfFailure=Sample field does not match Number definition in meta section
```

Comment:
If the file werre correct, it was a confusion. it was not in v4.1, I guess. I will add the files as failed_body_sample_011.vcf and leave a gap in v4.1, if that's ok.